app/src/main/java/com/github/libretube/ui/fragments/WatchHistoryFragment.kt

```diff
@@ -149,7 +149,8 @@ class WatchHistoryFragment : DynamicLayoutManagerFragment(R.layout.fragment_watc
 
         viewModel.filteredWatchHistory.observe(viewLifecycleOwner) { history ->
             binding.historyEmpty.isGone = history.isNotEmpty()
-            binding.historyContainer.isVisible = history.isNotEmpty()
+            binding.playAll.isEnabled = history.isNotEmpty()
```

Comment:
We should probably also disable the clear history button here?

Reply:
I thought about it, but I don't think so, since the clear button removes all items, and not just the visible ones. I guess it should rather be disabled, if the overall history is empty and not just the filtered one?